lua/lazyvim/plugins/extras/editor/fzf.lua

```diff
@@ -0,0 +1,142 @@
+---@class FzfLuaOpts: lazyvim.util.pick.Opts
+---@field cmd string?
+
+LazyVim.pick.commands = {
+  files = "files",
+}
+
+---@param command string
+---@param opts? FzfLuaOpts
+LazyVim.pick._open = function(command, opts)
+  opts = opts or {}
+
+  if opts.cmd == nil and command == "git_files" and opts.show_untracked then
+    opts.cmd = "git ls-files --exclude-standard --cached --others"
+  end
+  return require("fzf-lua")[command](opts)
+end
+
+return {
+  {
+    "nvim-telescope/telescope.nvim",
+    enabled = true,
+    keys = function()
+      return {}
+    end,
+  },
+  {
+    "ibhagwan/fzf-lua",
+    event = "VeryLazy",
+    opts = {
+      [1] = "default-title",
+      fzf_colors = {
+        ["fg"] = { "fg", "FzfLuaNormal" },
+        ["bg"] = { "bg", "FzfLuaNormal" },
+        ["hl"] = { "fg", "Special" },
+        ["fg+"] = { "fg", "Visual" },
+        ["bg+"] = { "bg", "Visual" },
+        ["hl+"] = { "fg", "Special" },
+        ["info"] = { "fg", "NonText" },
+        ["border"] = { "fg", "FzfLuaBorder" },
+        ["gutter"] = { "bg", "FzfLuaNormal" },
+        ["query"] = { "fg", "FzfLuaNormal" },
+        ["prompt"] = { "fg", "FzfLuaNormal" },
+        ["pointer"] = { "fg", "FzfLuaMarker" },
+        ["marker"] = { "fg", "FzfLuaMarker" },
+        ["header"] = { "fg", "FzfLuaNormal" },
+      },
+    },
+    config = function(_, opts)
+      local config = require("fzf-lua.config")
+      local actions = require("trouble.sources.fzf").actions
+      config.defaults.actions.files["ctrl-t"] = actions.open_all
+      config.defaults.actions.files["alt-t"] = actions.open
+      config.defaults.keymap.fzf["ctrl-q"] = "select-all+accept"
+      -- vim.env.FZF_DEFAULT_OPTS = ""
+      require("fzf-lua").setup(opts)
+    end,
```

Comment:
```suggestion
    opts = function()
      local config = require("fzf-lua.config")
      local actions = require("trouble.sources.fzf").actions
      config.defaults.actions.files["ctrl-t"] = actions.open_all
      config.defaults.actions.files["alt-t"] = actions.open
      config.defaults.keymap.fzf["ctrl-q"] = "select-all+accept"

      return {
        [1] = "default-title",
        fzf_colors = {
          ["fg"] = { "fg", "FzfLuaNormal" },
          ["bg"] = { "bg", "FzfLuaNormal" },
          ["hl"] = { "fg", "Special" },
          ["fg+"] = { "fg", "Visual" },
          ["bg+"] = { "bg", "Visual" },
          ["hl+"] = { "fg", "Special" },
          ["info"] = { "fg", "NonText" },
          ["border"] = { "fg", "FzfLuaBorder" },
          ["gutter"] = { "bg", "FzfLuaNormal" },
          ["query"] = { "fg", "FzfLuaNormal" },
          ["prompt"] = { "fg", "FzfLuaNormal" },
          ["pointer"] = { "fg", "FzfLuaMarker" },
          ["marker"] = { "fg", "FzfLuaMarker" },
          ["header"] = { "fg", "FzfLuaNormal" },
        },
      }
    end,
```
I could change a keymap in a `opts` function in personal configuration, but could not delete the keymap by setting its value to `nil` because it was being set up again in `config` function. I know you still probably have work to do and maybe you already intended to do that in the final form of this Extra, just thought I'd share my thoughts.

Reply:
Thanks!